versions/go.tex

```diff
@@ -264,8 +263,7 @@ \subsection{Geschäftsordnungsanträge}\label{geschuxe4ftsordnungsantruxe4ge}
   \item
     Verfahrensvorschlag
   \item
-    namentliche Abstimmung (ohne Gegenrede, ohne Abstimmung, setzt
-    Abstimmung per Handzeichen außer Kraft)
+  teilnamentliche Abstimmung (Fachschaften geben ihre Absicht namentlich abzustimmen vor der Gegenrede bekannt, Bedenkzeit möglich, bei Gegenrede sofort abgelehnt, schließt Abstimmung per Handzeichen und geheime Abstimmung aus, nach Annahme können Fachschaften bei der Redeleitung namentlich ihre Stimme abgeben) 
```

Comment:
Nach meiner Meinung ist dieses Verfahren eine auf mehreren Ebenen sehr schlechte Idee.

### Problem 1: Ein Werkzeug zur Unterdrückung

An keiner stelle wird hier erwähnt, dass nur Fachschaften ihre Stimme offenlegen dürfen, die dies auch am Anfang des Verfahrens angekündigt haben. Das bedeutet, dass eine Person die teilnamentliche Abstimmung beantragt, dann eine Fachschaft ankündigt, dass sie ihre Stimme namentlich abgeben möchte, und sobald es zur tatsächlichen Abstimmung kommt, können beliebig viele andere Fachschaften einen plötzlichen Meinungswechsel verspüren und doch noch ihre Stimme namentlich abgeben. Wenn das genug tun, kann damit zu Situationen kommen, in denen auch die Stimme der nicht namentlich abgegebenen Stimmen offen liegen. Alles, was ich auch schon im [ZaPF Wiki](https://zapf.wiki/Diskussion:WiSe24_AK_GO-%C3%84nderung:_Abstimmungsformen#Vorschlag_3a_-_GO-Antrag_zur_Möglichkeit_der_offenen_Abstimmung_für_einzelne_FSen) dazu geschrieben habe, bleibt also bestehen.

### Proplem 2: Dieses Verfahren wird nie zur Anwendung kommen

Da eine einzige Gegenrede reicht, das hier eingeführte Verfahren zu kippen, wird dies auch nie zum Einsatz kommen. Es wird sich immer eine einzelne Person finden, die sich gegen das Verfahren ausspricht. Die bessere Lösung wäre also wahrscheinlich, die namentliche Abstimmung einfach ersatzlos aus der GO zu streichen.

### Problem 3: Dieses Verfahren wird das Plenum künstlich in die Länge ziehen

Das hier vorgeschlagene Verfahren wird in der Praxis des Plenums eine halbe Ewigkeit dauern und selbst in der kürzest denkbaren Zeit zu ein paar Minuten pro Abstimmung in Anspruch nehmen, die zum Ende des Plenums extrem wertvoll sind um darüber zu entscheiden, ob ein Antrag noch behandelt werden kann oder nicht.

Nehmen wir mal in zwei Beispiele an, wie eine Abstimmung mit beantragter teilnamentlicher Abstimmung aussehen wird:

1. In diesem Fall sind alle im Plenum der Meinung, dass es eine gute Idee ist, einigen Fachschaften das Recht zu geben, die Stimmen anderer offenzulegen. Jemand stellt also diesen Antrag. Aus dem der Formulierung wird nicht ganz ersichtlich, wann es die Möglichkeit von Bedenkzeit gibt, aber ich nehme hier mal die sinnvollste Variante an und ermögliche Fachschaften darüber nachzudenken, ob sie ihre Stimme offenlegen möchten. Nach der Antragsstellung haben die Fachschaften also ein paar Minuten, sich darüber zu beraten. Dann geben eine Hand voll Fachschaften bekannt, dass sie dies tun möchten. Auf Nachfrage gibt es keine Gegenrede, die lauf Formulierung des Antrags auch erst hier überhaupt zum Ausdruck gebracht werden könnte, wir gehen also weiter im Verfahren. Nun versuchen Vertretende von Fachschaften sich durch das Plenum zur Redeleitung zu wuseln um ihre Stimmkarte und Stimme abzugeben. Daraufhin verliest die Redeleitung welche Fachschaft wie abgestimmt hat, aber erst nachdem sie es geschafft hat, wieder genug Ruhe ins Plenum zu bekommen. Im Anschluss dürfen alle anderen Fachschaften wie gewohnt durch Handzeichen abstimmen. Nach Bekanntgabe des finalen Ergebnisses bekommen die namentlich abstimmenden Fachschaften ihre Stimmkarten zurück und können sich setzen. Dabei entsteht erneut Unruhe, gegen die die Redeleitung ankämpfen muss.
2. Hier möchte jemand nicht, dass die teilnamentliche Abstimmung durchgeführt wird. Es wird also der GO-Antrag auf teilnamentliche Abstimmung gestellt und es kommt erst einmal zu einer Bedenkzeit um den Fachschaften die Möglichkeit zu geben, zu entscheiden ob sie namentlich abstimmen möchten oder nicht. Erst nachdem Fachschaften die Möglichkeit hatten, ihre Absicht kundzutun, kann das Verfahren durch eine Gegenrede hier abgebrochen werden. Es sind also mehrere Minuten für nichts und wieder nichts ins Land gegangen. Hier sei auch angemerkt, dass Fachschaften nur die Möglichkeit haben kundzutun, dass sie namentlich Abstimmen möchten, nicht warum. Die die Gegenrede stellende Person muss also durch die reine Präsenz dieser Fachschaften  davon überzeugt werden, keine Gegenrede zu stellen. Wird jedoch den Fachschaften die Möglichkeit geben, auch noch Gründe für die namentliche Abstimmung zu geben, könnte die Absichtserklärung nicht per Handzeichen geschehen, sondern müsste mündlich dargelegt werden. Das Verfahren zieht sich also noch weiter in die Länge.

Ich hoffe es wird klar, über welche Länge sich diesen Verfahren selbst im kürzesten Fall hinziehen wird. Dieser GO-Antrag wäre so auch ein mächtiges Werkzeug um mit ein Paar weiteren GO-Kniffen die bloße Behandlung unliebsamer Anträge gänzlich zu verhindern.

---

Das sind jetzt nur drei Problem, auf die ich in der Schnelle und ohne Diskussion mit anderen gekommen bin. In der Diskussion werdet ihr aber bestimmt noch weitere finden.